.circleci/config.yml

```diff
@@ -0,0 +1,17 @@
+version: 2.1
+orbs:
+  node: circleci/node@1.1.6
+jobs:
+  build-and-test:
+    executor:
+      name: node/default
+    steps:
+      - checkout
+      - node/with-cache:
+          steps:
+            - run: npm install
+            - run: npm test
+workflows:
+    build-and-test:
+      jobs:
+        - build-and-test
```

Comment:
I'm getting a little red icon in the diff telling me there isn't a newline at the end of the file, apparently they want that.